index.html

```diff
@@ -0,0 +1,2199 @@
+                    </a>
+                    <a href="#">
+                      <p class=" politics hero_stripe">Pandemic Heroes</p>
+                    </a>
+                  </div>
+                  <div class="hero-body">
+                    <a href="#">
+                      <h2 class="mag_link2 article-title py-2">
+                        COVID-19 Linked to Changes to the Brains of Some Patients
+                      </h2>
+                    </a>
+                    <p class="card-text card-text-hero pb-2">
+                      Have the pollsters learned from their mistakes in the 
+                      2016 presidential election? Maybe. Maybe not
+                    </p>
+                  </div>
+              </article>
+              <article class="col-sm-12 col-md-4 col-lg-4 w-100 border-0 d-flex flex-column">
+                <div>
+                  <a href="#">
+                    <img class=" rounded-0 hero-img w-100" src="https://d.newsweek.com/en/full/1578426/zoe-monterola-founder-six-feet-supplies.jpg" alt="image">
+                  </a>
+                  <a href="#">
+                    <p class=" politics hero_stripe">Pandemic Heroes</p>
+                  </a>
+                </div>
+                <div class="hero-body">
+                  <a href="#">
+                    <h2 class="mag_link2 article-title py-2">
+                      COVID-19 Linked to Changes to the Brains of Some Patients
+                    </h2>
+                  </a>
+                  <p class="card-text card-text-hero pb-2">
+                    Have the pollsters learned from their mistakes in the 
+                    2016 presidential election? Maybe. Maybe not
+                  </p>
+                </div>
+              </article>
+              <article class="col-sm-12 col-md-4 col-lg-4 w-100 border-0 d-flex flex-column">
+                <div>
+                  <a href="#">
+                    <img class=" rounded-0 hero-img w-100" src="https://d.newsweek.com/en/full/1578426/zoe-monterola-founder-six-feet-supplies.jpg" alt="image">
+                  </a>
+                  <a href="#">
+                    <p class=" politics hero_stripe">Pandemic Heroes</p>
+                  </a>
+                </div>
+                <div class="hero-body">
+                  <a href="#">
+                    <h2 class="mag_link2 article-title py-2">
+                      COVID-19 Linked to Changes to the Brains of Some Patients
+                    </h2>
+                  </a>
+                  <p class="card-text card-text-hero pb-2">
+                    Have the pollsters learned from their mistakes in the 
+                    2016 presidential election? Maybe. Maybe not
+                  </p>
+                </div>
+              </article>  
+            </div>
+          </section>
```

Comment:
You have this section duplicated on your page. On large screens,  you can see both.

If you want to show the sections in a different order depending on the screen size, this is not the way. There is an option called `order` (check it [here](https://css-tricks.com/almanac/properties/o/order/)) that you should use to arrange items (there's a class on bootstrap, too).